examples/basic_bq/variables.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "dataset_id" {
+  description = "Unique ID for the dataset being provisioned"
+}
+
+variable "dataset_name" {
+  description = "Friendly name for the dataset being provisioned"
+}
+
+variable "description" {
+  description = "Dataset description"
+}
+
+variable "expiration" {
+  description = "TTL of tables using the dataset in MS"
+}
+
+variable "project_id" {
+  description = "Project wheree the dataset and table are created"
+}
+
+variable "table_id" {
+  description = "Unique ID for the table being provisioned"
+}
+
+variable "region" {
```

Comment:
Blocker: The underlying `google_bigquery_dataset` uses the term [`location`](https://www.terraform.io/docs/providers/google/r/bigquery_dataset.html#location) which doesn't have a 1:1 mapping between normal [GCP regions](https://cloud.google.com/compute/docs/regions-zones/). I'm strongly of the opinion that we should just use the term `location` here - it doesn't overload the term `region` and involves less renaming and fewer surprises.